src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -352,4 +372,14 @@ private function changeQuantityOfOrderItem(string $orderItemId, int $quantity, s
 
         $this->cartsClient->executeCustomRequest($request);
     }
+
+    private function hasItemWithNameAndQuantity(Response $response, string $productName, int $quantity): bool
+    {
+        $items = json_decode($response->getContent(), true)['hydra:member'];
+
+        foreach ($items as $item) {
+            return ($item['productName'] === $productName) && $item['quantity'] === $quantity;
```

Comment:
Shouldn't you first find item with a given product name and only then check if it has a proper quantity? Because with this implementation, this method may not work with more than one item in cart